setup.py

```diff
@@ -26,6 +26,7 @@
         'widgetastic.core',
         'widgetastic.patternfly',
         'widgetastic.patternfly4',
+        'broker[docker]',
```

Comment:
Is it worth importing a whole new library just to execute the command on host? 

ssh2-python or paramiko could directly help.

Reply:
I wanted to avoid all the internal calls that we might need to do if we go the ssh2-python route. These are already abstracted by the broker.